proto/folio/taxed_reservation_nightly_quote.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package holms.types.folio;
+option csharp_namespace = "HOLMS.Types.Folio";
+option go_package = "folio";
+
+import "primitive/monetary_amount.proto";
+import "folio/tax_fee_quote.proto";
+import "primitive/pb_local_date.proto";
+
+message TaxedReservationNightlyQuote {
+	holms.types.primitive.PbLocalDate opsdate = 1;
+	holms.types.primitive.MonetaryAmount gross_lodging_charge = 2;
```

Comment:
Again...gross. I think this needs to be pretax.

It's telling that you don't use the same language for the quotes ("untaxed") and this line ("gross"), that makes me think we need different naming for this.

Reply:
That's definitely a better term. You're correct in assuming that I'm using "gross" as a pseudonym for pre-tax. I know gross is usually used when there's a subtractive change to a value that you want to ignore. This is my inner physicist acting out. We don't believe subtraction really exists.